api/src/main/java/org/openmrs/module/diagnosis/DiagnosisService.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.module.diagnosis;
+
+import org.openmrs.Encounter;
+import org.openmrs.Obs;
+import org.openmrs.Patient;
+import org.openmrs.api.OpenmrsService;
+import org.openmrs.Diagnosis;
+
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * <pre>
+ * API methods for managing diagnoses
+ * </pre>
+ */
+public interface DiagnosisService extends OpenmrsService {
+
+	/**
+	 * Changes a non-coded diagnosis to coded diagnosis
+	 * 
+	 * @param nonCodedObs an Obs that contains the non-coded diagnosis
+	 * @param diagnoses a List of Diagnosis representing the new diagnoses
+	 * @return
+	 */
+	List<Obs> codeNonCodedDiagnosis(Obs nonCodedObs, List<Diagnosis> diagnoses);
```

Comment:
For now, let us get rid of the above method

Reply:
Done